DEVELOPMENT.md

```diff
@@ -26,3 +26,15 @@ Publish the draft release
 If you're debugging a running server than `IO.inspect` is a good approach, any messages you create with it will be sent to your LSP client as a log message
 
 To debug in tests you can use `IO.inspect(Process.whereis(:user), message, label: "message")` to send your output directly to the group leader of the test process.
+
+# Documentation website
```

Comment:
Is there a minimum python version needed for this?

Reply:
At least Python 3, mentioned in the doc now. As long as Mkdocs works.